fragments/infra-boot.sh

```diff
@@ -242,7 +242,7 @@ else
         notify_failure "could not install pyOpenSSL"
 
     # Install from the EPEL repository
-    yum -y --enablerepo=epel install ansible ||
+    yum -y --enablerepo=epel install ansible1.9 ||
```

Comment:
Ideally we should remove external repos completely (especially on RHEL hosts). I wonder if we still need epel version of ansible? I'm not even sure what was the reason before to depend on epel version (nothing descriptive in logs).